<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAc</class>
 <widget class="QDialog" name="DialogAc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1141</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Nom de l'activite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>260</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Domaine</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_idAc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nomAc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>180</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>10</y>
       <width>1201</width>
       <height>771</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../ATELIE~1/JOUETS~1.JPG</pixmap>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeDebut">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>320</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeFin">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>380</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>325</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>date debut </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>date fin</string>
     </property>
    </widget>
    <zorder>label_8</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>le_idAc</zorder>
    <zorder>le_nomAc</zorder>
    <zorder>ajouter</zorder>
    <zorder>comboBoxAc</zorder>
    <zorder>dateTimeDebut</zorder>
    <zorder>dateTimeFin</zorder>
    <zorder>label_13</zorder>
    <zorder>label_14</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QTableView" name="tableac">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>31</y>
       <width>851</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAfficher">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>80</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>-90</x>
       <y>10</y>
       <width>1231</width>
       <height>611</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../ATELIE~1/JOUETS~1.JPG</pixmap>
     </property>
    </widget>
    <zorder>label_9</zorder>
    <zorder>tableac</zorder>
    <zorder>btnAfficher</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Nom de l'activite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>300</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Domaine</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifacid">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>161</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="modifacnom">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>230</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxmodifdomaine">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>290</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnmodifAc">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>220</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>0</y>
       <width>1171</width>
       <height>671</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../ATELIE~1/JOUETS~1.JPG</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>360</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Date Debut</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>420</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Date Fin</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="ModdateTimeDebut">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>360</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="ModdateTimeFin">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>420</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <zorder>label_10</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>modifacid</zorder>
    <zorder>modifacnom</zorder>
    <zorder>comboBoxmodifdomaine</zorder>
    <zorder>btnmodifAc</zorder>
    <zorder>label_15</zorder>
    <zorder>label_16</zorder>
    <zorder>ModdateTimeDebut</zorder>
    <zorder>ModdateTimeFin</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>ID Activite</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="suppAc">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>190</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnsupAc">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>290</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>1111</width>
       <height>801</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../ATELIE~1/JOUETS~1.JPG</pixmap>
     </property>
    </widget>
    <zorder>label_11</zorder>
    <zorder>label_7</zorder>
    <zorder>suppAc</zorder>
    <zorder>btnsupAc</zorder>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>recherche</string>
    </attribute>
    <widget class="QTableView" name="rechAc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>1091</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rechAc_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rechercheAc">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triNom">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triD">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>Domaine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triId">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;
border-color: rgb(197, 39, 68);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(225, 45, 78));
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(217, 29, 81, 255));
selection-color: rgb(225, 22, 69);
selection-background-color: rgb(225, 6, 57);</string>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>10</y>
       <width>1211</width>
       <height>561</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../ATELIE~1/JOUETS~1.JPG</pixmap>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>rechAc</zorder>
    <zorder>rechAc_2</zorder>
    <zorder>rechercheAc</zorder>
    <zorder>triNom</zorder>
    <zorder>triD</zorder>
    <zorder>triId</zorder>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>calendrier</string>
    </attribute>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>471</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableCal">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>561</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>-110</x>
       <y>20</y>
       <width>1221</width>
       <height>631</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../ATELIE~1/JOUETS~1.JPG</pixmap>
     </property>
    </widget>
    <zorder>label_17</zorder>
    <zorder>calendarWidget</zorder>
    <zorder>tableCal</zorder>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Page</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
